### PR TITLE
Skip checking existence of trailing comment in `CommaRule` if already violating

### LINKED ISSUE #683: Regression in CommaRule from 0.9.1 introduced in 0.9.2
Hello! With swiftlint 0.9.1, the following source listing correctly produces a `CommaRule` violation. As of swiftlint 0.9.2, it no longer produces the violation.

```swift
func plus(first: Int, second: Int) -> Int {
	return first + second
}

let result = plus(
	first: 3 , // <--- this should produce an error
	second: 4
)
```

cc @norio-nomura

### PR DESCRIPTION
**Pull request by norio-nomura**
Fix #683

### CODE REVIEW

File: Source/SwiftLintFramework/Rules/CommaRule.swift [hunk 1]
```diff
@@ -111,6 +112,12 @@ public struct CommaRule: CorrectableRule, ConfigurationProviderRule {
                     return nil
                 }
 
+                // If the first range does not start with comma, it already violates this rule
+                // whatever is contained in the second range.
```
Comment (jpsim) [inline Source/SwiftLintFramework/Rules/CommaRule.swift]: "no matter what is contained in the second range."

File: CHANGELOG.md [hunk 2]
```diff
@@ -41,6 +41,10 @@
   [Andrew Rahn](https://github.com/paddlefish)
   [#667](https://github.com/realm/SwiftLint/issues/667)
 
+* Fix `CommaRule` ignored some violations when trailing comment exists.
```
Comment (jpsim) [inline CHANGELOG.md]: How about this?

> Fix regression in `CommaRule` ignoring violations when the comma is followed
> by a new line.

**Comment by jpsim** [comment]
other than minor wording suggestions 👍

File: CHANGELOG.md (hunk 2 above)
Comment (norio-nomura) [inline CHANGELOG.md]: The issue is not caused by a new line but by comment.

File: Source/SwiftLintFramework/Rules/CommaRule.swift (hunk 1 above)
Comment (norio-nomura) [inline Source/SwiftLintFramework/Rules/CommaRule.swift]: Thanks!

File: CHANGELOG.md (hunk 2 above)
Comment (norio-nomura) [inline CHANGELOG.md]: Anyway, thanks!

**Comment by norio-nomura** [comment]
Updated with `[ci skip]` in commit message. :pray:

**Comment by jpsim** [comment]
👍 nicely done!

**Comment by norio-nomura** [comment]
Thanks! :pray:

**Pull request merged**